service/controller/release/resource/status/create.go

```diff
@@ -0,0 +1,88 @@
+package status
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	releasev1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller/context/reconciliationcanceledcontext"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/release-operator/service/controller/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToReleaseCR(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	var releaseCycleCR *releasev1alpha1.ReleaseCycle
+	{
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding ReleaseCycle CR %#q in namespace %#q", cr.Name, cr.Namespace))
+
+		// Release CR and corresponding ReleaseCycle CR have the same name and namespace.
+		releaseCycleCR, err = r.g8sClient.ReleaseV1alpha1().ReleaseCycles(cr.Namespace).Get(cr.Name, metav1.GetOptions{})
```

Comment:
```suggestion
		releaseCycleCR, err = r.g8sClient.ReleaseV1alpha1().ReleaseCycles(cr.GetNamespace()).Get(cr.GetName(), metav1.GetOptions{})
```

Reply:
Are you big on that / have a reason to use getter methods here? I personally prefer using fields directly. Less typing less parenthesis looks a bit clearer.

Reply:
Not big on it. But they are there for a reason.

Reply:
I think there reason for them to be is fulfilling interfaces which makes sense. I don't feel like they are intended to be used everywhere. I can be wrong though.